src/components/importwms/ImportWmsDirective.js

```diff
@@ -43,14 +43,15 @@ goog.require('ga_wms_service');
               // Kill the current uploading
               $scope.cancel();
 
-              var proxyUrl = gaUrlUtils.proxifyUrl(url);
+              url = gaUrlUtils.isAdminValid(url) && gaUrlUtils.isHttps(url) ?
+                  url : gaUrlUtils.proxifyUrl(url);
```

Comment:
I don't see any difference. In proxifyUrl, we already have these tests. What urls gets a proxy that shouldn't?